docs/_includes/browsers.md

```diff
@@ -1,6 +1,6 @@
 There is a lot of nuance in browser support, but in general, Fluid Framework supports modern browsers, and we recommend
 that you use the most up-to-date browser that's compatible with your operating system. The following browsers are
-supported:
+supported.
```

Comment:
Is it always preferred to avoid the colon, even when using "the following" to introduce a list? (Asking to learn, not to block this change.)

Reply:
It depends. Per the [MS Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/f/following), you should end with a period unless "following" is the last word before the list it introduces, in which case you would use a colon instead.